src/components/20-molecules/dropdown/ui.test.js

```diff
@@ -6,48 +6,41 @@ fixture('Dropdown').page(`${host}/iframe.html?id=molecules-dropdown--dropdown`);
 
 test('should render correctly', async t => {
   const dropdown = await Selector(() =>
-    document.querySelector(`axa-dropdown[data-test-id="dropdown"]`)
+    document.querySelector(`axa-dropdown`)
   );
   await t.expect(dropdown.exists).ok();
 });
 
-fixture('Dropdown valid').page(
-  `${host}/iframe.html?id=molecules-dropdown--dropdown-w-valid-checkmark`
+fixture('Dropdown check mark').page(
+  `${host}/iframe.html?id=molecules-dropdown--dropdown&knob-checkmark=true`
 );
 test('should show checkmark', async t => {
   const dropdownCheckmark = await Selector(() =>
     document.querySelector(
-      `axa-dropdown[data-test-id="dropdown-valid"] .m-dropdown__valid-icon .m-dropdown__valid-icon-inner-active`
+      `axa-dropdown .m-dropdown__check`
     )
   );
   await t.expect(dropdownCheckmark.exists).ok();
 });
 
 fixture('Dropdown error').page(
-  `${host}/iframe.html?id=molecules-dropdown--dropdown-w-error-message`
+  `${host}/iframe.html?id=molecules-dropdown--dropdown&knob-invalid=true&knob-error=error`
 );
 
-test('should show error message and correct colors', async t => {
-  const dropdown = await Selector(() =>
-    document.querySelector(`axa-dropdown[data-test-id="dropdown-error"]`)
+test('should show error message and have the correct color', async t => {
+  const $axaError = await Selector(() =>
+    document.querySelector(`axa-dropdown .m-dropdown__error `)
   );
-
-  await t.expect(dropdown.exists).ok();
-
-  const getErrorMessage = ClientFunction(() => {
-    const errorDiv = document.querySelector(
-      'axa-dropdown[data-test-id="dropdown-error"] > .m-dropdown__error'
-    );
-    return errorDiv.innerText;
-  });
-
-  await t.expect(await getErrorMessage()).eql('please select an item');
+  await t.expect($axaError.innerText).eql('error');
+  await t
+    .expect(await $axaError.getStyleProperty('color'))
+    .eql('rgb(201, 20, 50)');
 
   const getBorderColor = ClientFunction(() => {
     return window
       .getComputedStyle(
         document.querySelector(
-          'axa-dropdown[data-test-id="dropdown-error"] .m-dropdown__list--enhanced'
+          'axa-dropdown .m-dropdown__toggle'
```

Comment:
I would recommend using `js-dropdown__toggle` instead of `m-dropdown__toggle`

Reply:
good catch! Done.